plugins/single-instance/README.md

```diff
@@ -49,6 +49,8 @@ fn main() {
 }
 ```
 
+Note that currently, plugins run in the order they were added in to the builder, you need to register this plugin first if you don't want the other plugins to run on the new instance
```

Comment:
```suggestion
Note that currently, plugins run in the order they were added in to the builder, you need to register this plugin first if you don't want the other plugins to run on the temporary instances.
```

really not sure about this one, "new" sounds wrong to me. Temporary isn't much better though, maybe secondary? (secondary is what i use in discussions on discord at least but idk if it's good)

Reply:
what about `non-main instances`? they all sound weird if there's no context explaining how this thing actually works to be honest

Reply:
We can also leave out the explanation (for now?) and just do ```
Note that currently, plugins run in the order they were added in to the builder, so make sure that this plugin is registered first.
```